docs/domains/Domain.json

```diff
@@ -200,11 +213,12 @@
       "description": "DomainSpec is a description of a domain.",
       "type": "object",
       "properties": {
-        "keepJRFSchema": {
-          "description": "Keep JRF schema between lifecycle updates",
-          "type": "boolean"
+        "configuration": {
+          "description": "Models and overrides affecting the WebLogic domain configuration",
+          "$ref": "#/definitions/Configuration"
         },
         "configOverrideSecrets": {
+          "deprecated": "true",
           "description": "A list of names of the secrets for optional WebLogic configuration overrides.",
```

Comment:
For deprecated fields, it looks like it'd be helpful to update its description to include the location of the field that replaces it...